Forms/frmMain.cs

```diff
@@ -75,40 +93,79 @@ private void updateHostsList(JsonResponse fetchedRecords)
-                row.SubItems.Add(fetchedRecords.response.recs.objs[i].display_name);
-                row.SubItems.Add(fetchedRecords.response.recs.objs[i].display_content);
-
+                row.SubItems.Add(r.type);
+                row.SubItems.Add(r.name);
+                row.SubItems.Add(r.content);
+                row.SubItems.Add(r.modified_on.ToShortDateString() + " " + r.modified_on.ToShortTimeString());
                 //Only check this if it's an A record. MX records may have the same name as the primary A record, but should never be updated with an IP.
-                if ((Array.IndexOf(selectedHosts, fetchedRecords.response.recs.objs[i].display_name) >= 0) == true && fetchedRecords.response.recs.objs[i].type == "A")
+                bool NeedIp = false;
+                switch (r.type)
                 {
-                    row.Checked = true;
-                }
+                    case "A":
+                        NeedIp = true;
+                        break;
 
-                listViewRecords.Items.Add(row);
+                    case "AAAA":
+                        NeedIp = true;
+                        break;
+                }
+                if (NeedIp)
+                {
+                    int pos = Array.IndexOf(selectedHosts, row.SubItems[2].Text);
+                    if (pos > -1)
+                    {
+                        row.Checked = true;
+                    }
+                }
+                try
+                {
+                    if (Program.settingsManager.getSetting("HideSRV").ToBool())
+                    {
+                        if(r.type != "SRV")
+                        {
+                            listViewRecords.Items.Add(row);
+                        }
+                    }
+                    else
+                    {
+                        listViewRecords.Items.Add(row);
+                    }
+                }
+                catch (Exception)
+                {
+                    listViewRecords.Items.Add(row);
+                }
             }
 
             //Grey out anything that isn't an A record.
-            for(int j = 0; j < listViewRecords.Items.Count; j++)
+            for (int j = 0; j < listViewRecords.Items.Count; j++)
             {
-                if (listViewRecords.Items[j].SubItems[1].Text != "A")
+                bool NeedIp = false;
+                switch (listViewRecords.Items[j].SubItems[1].Text)
+                {
+                    case "A":
+                        NeedIp = true;
+                        break;
+
+                    case "AAAA":
+                        NeedIp = true;
+                        break;
+                }
+                if (NeedIp == false)
                 {
-                    listViewRecords.Items[j].ForeColor = System.Drawing.Color.LightGray;
+                    listViewRecords.Items[j].ForeColor = System.Drawing.Color.Gray;
                 }
             }
-

```

Comment:
I'm not against removing this whitespace, but:
1) If they are being removed, remove them everywhere.
2) This makes the PR bigger than it needs to be.